deployment/helm/recommendation-service/requirements.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: neo4j
```

Comment:
@paulczar @michael-simons How do we disable auth by default in the Neo4j helm chart?

Reply:
if you check `deployment/helm/recommendation-service/values.yaml` you'll see the following set which disables auth.

```
neo4j.authEnabled: false
```